src/mame/drivers/cischeat.cpp

```diff
@@ -1184,13 +1185,17 @@ static INPUT_PORTS_START( bigrun )
 	PORT_DIPSETTING(    0x02, "2 (Green-White Car)" )
 	PORT_DIPSETTING(    0x04, "3 (Red-White Car)" )
 	PORT_DIPSETTING(    0x06, "4 (Yellow Car)" )
-	PORT_DIPUNKNOWN_DIPLOC(  0x08, 0x00, "SW1:4")       /* Always ON according to manual */
+	/* The manual states SW1:4 always set to ON when linking games but doesn't mention if
+	setting SW1:4 OFF disables linking */
+	PORT_DIPNAME( 0x08, 0x00, "Enable Linking" ) PORT_DIPLOCATION("SW1:4")
+	PORT_DIPSETTING(    0x00, DEF_STR( Yes ) )
+	PORT_DIPSETTING(    0x08, DEF_STR( No ) )
```

Comment:
The comment here doesn’t exactly inspire confidence.  You say the manual “doesn’t mention if setting SW1:4 OFF disables linking” – what are you basing the change in description on?  Have you confirmed that’s all it does from the code?